src/containers/reservations/Accordion.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useRef } from 'react';
```

Comment:
Is accordion a container? A container is typically rendered directly to a route.

Reply:
I'm not sure what you mean by "container" (is it "an informal term for a React component that is `connect`ed to a redux store"?) or "rendered directly to a route" (as opposed to what?) here. The Accordion component contains the "slide down" functionality and its children are the contents of the slide down. Are you saying that the Accordion component code should be moved to a different file?